src/NServiceBus.Persistence.AzureTable/NServiceBus.Persistence.AzureTable.csproj

```
@@ -11,7 +11,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="[1.0.8, 2.0.0)" />
-    <PackageReference Include="NServiceBus" Version="8.0.0-alpha.1900" />
+    <PackageReference Include="NServiceBus" Version="8.0.0-beta.1" />
```

should this be a version range?
```suggestion
    <PackageReference Include="NServiceBus" Version="[8.0.0-beta.1, 9.0.0)" />
```